dev/checkstyle.xml

```diff
@@ -190,6 +190,10 @@
             <property name="format" value="new URL\("/>
             <property name="message" value="Use URI.toURL or URL.of instead of URL constructors." />
         </module>
+        <module name="RegexpSinglelineJava">
```

Comment:
We should ban the use of `org.apache.commons.lang.builder.ToStringBuilder` simultaneously, even though it is not currently being used, because the `commons-lang/2.6//commons-lang-2.6.jar` is still a dependency of Spark.